config/environments/production.rb

```diff
@@ -54,6 +54,7 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
+  config.ssl_options = { redirect: { exclude: ->(request) { request.path.include?('api/v1/status') } } }
```

Comment:
This sounds like a very specific use case associated with the infrastructure of your project.
Why we need this now? We were able to deploy many apps with the existing config without any issue.

Reply:
@JulianPasquale All internal apps that we have disable the `force_ssl` to skip this issue